front/src/View/Pages/Personal.tsx

```diff
@@ -20,27 +20,51 @@ class Personal extends React.Component<any, PersonalState> {
         this.fillIcon = this.fillIcon.bind(this);
         this.getIcon = this.getIcon.bind(this);
         this.renderLobbies = this.renderLobbies.bind(this);
+        this.delete = this.delete.bind(this);
+        this.deleteLobby = this.deleteLobby.bind(this);
+        this.refreshData = this.refreshData.bind(this);
     }
 
     public componentDidMount(): void {
+        this.refreshData();
+    }
+
+    public delete(data: any): void {
+        this.refreshData();
+    }
+
```

Comment:
In **delete()** function I can't see the utility of the parameters (data:any) if we don't use it.
And.. maybe I'm wrong but **componentDidMount()** and **delete()** have the same utility no ? maybe just one function is more appropriate

Reply:
Right, it is temporary until "real" front is done.
No, cpntDidMount initiates data, and delete updates data.